bin/pulumilocal

```diff
@@ -349,44 +341,62 @@ def get_service_endpoint():
     return endpoint
 
 
-def prepare_config_file():
-    file_name = 'Pulumi.%s.yaml' % PULUMI_STACK_NAME
-    if os.path.exists(file_name):
-        return file_name
-
-    # construct endpoints config
-    endpoint = get_service_endpoint()
-    endpoints = DEFAULT_CONFIG['config']['aws:endpoints']
-    if not endpoints:
-        entry = {service: endpoint for service in SERVICES}
-        endpoints.append(entry)
-
-    # write local config file
-    with open(file_name, 'w') as fp:
-        fp.write(yaml.dump(DEFAULT_CONFIG))
-    return file_name
-
-
-def prepare_cmd_args():
-    """
-    Constructs command line arguments for running the "pulumi" CLI
-    """
-    cmd_args = list(sys.argv)
-    if 'up' in cmd_args and '-s' not in cmd_args:
-        cmd_args += ['-s', PULUMI_STACK_NAME]
-    return cmd_args
+def set_localstack_pulumi_config(args: argparse.Namespace):
+    # LocalStack Endpoint
+    service_url = get_service_endpoint()
+    # Create argument list to pulumi config set-all
+    config_args = list()
+    config_args.append(PULUMI_CMD)
+    config_args.append("config")
+    if args.stack is not None:
+        config_args.append("--stack")
+        config_args.append(args.stack)
+    if args.cwd is not None:
+        config_args.append("--cwd")
+        config_args.append(args.cwd)
+    config_args.append("set-all")
+    config_args.append("--plaintext")
+    config_args.append("aws:region=us-east-1")
```

Comment:
nitpick: we could define these configs in a global list or dict, and then add them via `config_args.append(..)` in a `for` loop here. (not critical, though - shouldn't hold back the merge)